deepchecks/tabular/metric_utils/scorers.py

```diff
@@ -117,8 +129,11 @@ def __init__(self, scorer: t.Union[str, t.Callable], name: str = None):
         if isinstance(scorer, str):
             formated_scorer_name = scorer.lower().replace('sensitivity', 'recall').replace('specificity', 'tnr')\
                 .replace(' ', '_')
-            if formated_scorer_name in _func_dict:
-                self.scorer = _func_dict[formated_scorer_name]
+            if formated_scorer_name in _regression_scorers_dict and not formated_scorer_name.startswith('neg'):
```

Comment:
IMO this condition is not general enough, because in the future we might add a metric that support our convention but doesn't start with "neg" or change a metric string name for some reason.